app/controllers/support_interface/ucas_matches_controller.rb

```diff
@@ -54,12 +54,5 @@ def record_ucas_withdrawal_requested
       flash[:success] = 'The date of requesting withdrawal from UCAS was recorded'
       redirect_to support_interface_ucas_match_path(match)
     end
-
-    def process_match
```

Comment:
We can also remove the route 🚮 
https://github.com/DFE-Digital/apply-for-teacher-training/blob/master/config/routes.rb#L701